draft-ietf-quic-transport.md

```diff
@@ -4739,8 +4747,9 @@ server MAY send multiple Initial packets.  The cryptographic key exchange could
 require multiple round trips or retransmissions of this data.
 
 The payload of an Initial packet includes a CRYPTO frame (or frames) containing
-a cryptographic handshake message, ACK frames, or both.  PING, PADDING, and
-CONNECTION_CLOSE frames of type 0x1c are also permitted.  An endpoint that
+a cryptographic handshake message, ACK frames, CONNECTION_CLOSE frames of type
+0x1c, or some combination thereof.  PING and PADDING frames are also permitted.
+An endpoint that
```

Comment:
This is pretty awkward.  The idea is that the old text implies that it is invalid to have an Initial with neither a CRYPTO nor an ACK.  But IIUC it's valid to send CONNECTION_CLOSE and nothing else...

Reply:
Done.